config/default.yaml

```diff
@@ -8,6 +8,8 @@ services:
     tokenURL: 'https://idam-api.aat.platform.hmcts.net/o/token'
     clientID: 'divorce'
     clientSecret: 'NEED TO INSERT SECRET'
+  cos:
+    baseURL: 'http://localhost:4012'
```

Comment:
Do we need to add some instructions to the readme about how to get this service running locally?

Reply:
included this however, you just need to run cos, nothing special config needed, let me know if i need to include more.